packages/arb-token-bridge-ui/tests/support/common.ts

```diff
@@ -23,17 +24,47 @@ export const goerliRPC = `https://goerli.infura.io/v3/${Cypress.env(
 )}`
 export const arbitrumGoerliRPC = 'https://goerli-rollup.arbitrum.io/rpc'
 
+export const ERC20TokenAddressL1 = '0x326C977E6efc84E512bB9C30f76E30c160eD06FB'
+export const ERC20TokenAddressL2 = '0xd14838A68E8AFBAdE5efb411d5871ea0011AFd28'
+
+export const zeroToLessThanOneETH = /0(\.\d+)*( ETH)/
+export const zeroToLessThanOneERC20 = /0(\.\d+)*( LINK)/
+
+export const gasConfig = { gasFee: 100, gasLimit: 1000000 } // a high gas config to ensure transactions ALWAYS succeed
+
 export async function getInitialETHBalance(rpcURL: string): Promise<BigNumber> {
   const provider = new JsonRpcProvider(rpcURL)
   return await provider.getBalance(Cypress.env('ADDRESS'))
 }
 
-export const setupMetamaskNetwork = (networkType: NetworkType) => {
+export async function getInitialERC20Balance(
+  tokenAddress: string,
+  rpcURL: string
+): Promise<BigNumber> {
+  const provider = new JsonRpcProvider(rpcURL)
+  const multiCaller = await MultiCaller.fromProvider(provider)
+  const [tokenData] = await multiCaller.getTokenData([tokenAddress], {
+    balanceOf: { account: Cypress.env('ADDRESS') },
+    decimals: true
```

Comment:
If we're not using the `decimals` info, we can omit it from the params

Reply:
Done